cmd/main.go

```diff
@@ -33,6 +33,7 @@ var (
 	probeAddr, metricsAddr string
 	log                    = logger.Log
 	cf                     *config.NSXOperatorConfig
+	ncpNamespace           = "default"
```

Comment:
call it namespace or nsxOperatorNamespace

Reply:
ok